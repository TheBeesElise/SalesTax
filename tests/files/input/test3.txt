1 encyclopedia set at 199.99
1 imported CD at 19.99
2 music anthology at 99.99
3 bag of imported chips at 4.99
5 egg at 0.10
