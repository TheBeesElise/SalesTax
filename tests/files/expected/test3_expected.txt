1 encyclopedia set: $200.00
1 imported CD: $23.00
2 music anthology: $220.00
3 bag of imported chips: $15.75
5 egg: $0.50
Sales Taxes: $23.75
Total: $459.25
